database/cassandraSeed.js

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable no-console */
+const faker = require('faker');
+const fs = require('file-system');
+const path = require('path');
+
+// Number of records
+const numberOfListings = 1000000;
```

Comment:
create 10 million listings but you have 1 million written. Ignore if intentional.